src/persistence/settings.cpp

```diff
@@ -1786,6 +1786,16 @@ Settings::Request Settings::getFriendRequest(int index) const
     return friendRequests.at(index);
 }
 
+void Settings::deleteFriendRequest(const QString &stringId)
```

Comment:
This commit is a bit hidden in the middle of a const-correctness pull-request. It's not okay since it's at least in the commit name, but please try to mention it in the description of the pull request if you do some other things.